<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordEntryWidget</class>
 <widget class="QWidget" name="WordEntryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Translated word</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="valueEdit">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Disambiguation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="disambiguationEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Optional: Use this to distinguish homonyms.&lt;br/&gt;ie. for the word &amp;quot;fly&amp;quot; you could specify the disambiguation &amp;quot;insect&amp;quot;.&lt;br/&gt;&lt;br/&gt;Will show up as &lt;span style=&quot; font-weight:700;&quot;&gt;fly (insect)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="detailsEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
